reflex/components/radix/primitives/drawer.py

```diff
@@ -33,14 +33,29 @@ class DrawerRoot(DrawerComponent):
 
     alias = "Vaul" + tag
 
+    # The open state of the drawer when it is initially rendered. Use when you do not need to control its open state.
+    default_open: Var[bool]
+
     # Whether the drawer is open or not.
     open: Var[bool]
 
-    # Enable background scaling, it requires an element with [vaul-drawer-wrapper] data attribute to scale its background.
-    should_scale_background: Var[bool]
+    # Fires when the drawer is opened.
+    on_open_change: EventHandler[identity_event(bool)]
 
-    # Number between 0 and 1 that determines when the drawer should be closed.
-    close_threshold: Var[float]
+    # When `False`, it allows to interact with elements outside of the drawer without closing it. Defaults to `True`.
+    modal: Var[bool]
+
+    # Direction of the drawer. This adjust the animations and the drag direction. Defaults to `"bottom"`
+    direction: Var[LiteralDirectionType]
+
+    # Gets triggered after the open or close animation ends, it receives an open argument with the open state of the drawer by the time the function was triggered.
+    on_animation_end: EventHandler[identity_event(bool)]
+
+    # When false dragging, clicking outside, pressing esc, etc. will not close the drawer. Use this in combination with the open prop, otherwise you won't be able to open/close the drawer.
```

Comment:
```suggestion
    # When `False`, dragging, clicking outside, pressing esc, etc. will not close the drawer. Use this in combination with the open prop, otherwise you won't be able to open/close the drawer.
```

Reply:
fixed